pkg/ssh/ssh.go

```diff
@@ -303,33 +299,29 @@ func NewSSHTunnelList(user, keyfile string, healthCheckURL *url.URL, stopChan ch
 	go wait.Until(func() {
 		l.tunnelsLock.Lock()
 		defer l.tunnelsLock.Unlock()
-		// Healthcheck each tunnel every minute
-		numTunnels := len(l.entries)
-		for i, entry := range l.entries {
-			// Stagger healthchecks evenly across duration of healthCheckPoll.
-			delay := healthCheckPoll * time.Duration(i) / time.Duration(numTunnels)
-			l.delayedHealthCheck(entry, delay)
+		for address, tunnel := range l.tunnels {
+			l.delayedHealthCheck(address, tunnel, wait.Jitter(healthCheckPoll, 1))
 		}
 	}, healthCheckPoll, stopChan)
 	return l
 }
 
-func (l *SSHTunnelList) delayedHealthCheck(e sshTunnelEntry, delay time.Duration) {
+func (l *SSHTunnelList) delayedHealthCheck(address string, tunnel tunnel, delay time.Duration) {
 	go func() {
 		defer runtime.HandleCrash()
 		time.Sleep(delay)
-		if err := l.healthCheck(e); err != nil {
-			klog.Errorf("Healthcheck failed for tunnel to %q: %v", e.Address, err)
-			klog.Infof("Attempting once to re-establish tunnel to %q", e.Address)
-			l.removeAndReAdd(e)
+		if err := l.healthCheck(tunnel); err != nil {
+			klog.Errorf("health check failed for tunnel to %q: %v", address, err)
```

Comment:
Logs should be capitalized, just not error strings:

https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
fixed.